src/tests/simulation/src/slt.rs

```diff
@@ -59,16 +70,31 @@ impl SqlCmd {
 }
 
 fn extract_sql_command(sql: &str) -> SqlCmd {
-    let cmd = sql
-        .trim_start()
-        .split_once(' ')
-        .unwrap_or_default()
-        .0
-        .to_lowercase();
-    match cmd.as_str() {
-        "create" => SqlCmd::Create {
-            is_create_table_as: is_create_table_as(sql),
+    let mut tokens = sql.split_whitespace().map(|s| s.to_lowercase());
+    fn next_token(tokens: &mut impl Iterator<Item = String>) -> String {
+        tokens.next().unwrap()
```

Comment:
If we have some tests intended to fail with parse error, will this lead to panic of the test driver?